admin_guide/integrations.rst

```diff
@@ -0,0 +1,336 @@
+External Integrations
+#####################
+
+Ingesting Data from Kafka
+=========================
+
+Citus can leverage existing Postgres data ingestion tools. For instance, we can use a tool called `kafka-sink-pg-json <https://github.com/justonedb/kafka-sink-pg-json>`_ to copy JSON messages from a Kafka topic into a database table. As a demonstration, we'll create a ``kafka_test`` table and ingest data from the ``test`` topic with a custom mapping of JSON keys to table columns.
+
+The easiest way to experiment with Kafka is using the Confluent platform, which includes Kafka, Zookeeper, and associated tools whose versions are verified to work together.
```

Comment:
Maybe link to the platform here? https://www.confluent.io/product/confluent-platform/